less/about/newtab.less

```diff
@@ -20,14 +20,13 @@ body {
   background: #fff;
 }
 
-body, .dynamicBackground {
+body, .dynamicBackground, bgGradient {
   opacity: 0;
   animation: fadeIn 200ms;
   animation-fill-mode: forwards;
 }
 
 .copyrightNotice {
-  visibility: hidden !important;
```

Comment:
This is no longer needed because the FooterInfo component checks for (and hides) copyright info if no background is shown